### PR TITLE
Improving documentation about creating Sharded Keyspace. 

### LINKED ISSUE #1434: Documentation about creating Sharded Keyspace should be clearer
Reading the Documentation about sharded keyspace 
https://vitess.io/docs/16.0/user-guides/vschema-guide/sharded/ 
leaves the impression that it is enough to create shards for a table when executing the json via 
`vtctldclient ApplyVSchema --vschema-file vschema_customer_sharded.json customer`
But after "just" doing that and executing
```
mysql> use `customer:-80`;
Database changed
mysql> select * from customer;
```
the error of 
`ERROR 1105 (HY000): target: customer.-80.primary: no healthy tablet available for 'keyspace:"customer" shard:"-80" tablet_type:PRIMARY'` 
occurs. 

All steps from the scripts (30X) are necessary to create shards. The documentation is clearly not saying that.

### PR DESCRIPTION
**Pull request by Satarupa22-SD**
Adding this section would simplify the process for the users. 
Resolves #1434

### CODE REVIEW

**Comment by Satarupa22-SD** [comment]
@mattlord please look into this. Thanks.

File: content/en/docs/20.0/user-guides/vschema-guide/sharded.md
Comment (mattlord) [inline content/en/docs/20.0/user-guides/vschema-guide/sharded.md:1]: This has to be done in 21.0 docs at a minimum, as that is what will become 22.0 ...

File: content/en/docs/20.0/user-guides/vschema-guide/sharded.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+3. Create physical shards
+4. Create tablets (instances) for each shard
+5. Initialize the schema on each shard
+6. (Optional) Migrate data from an unsharded keyspace
+
+Each of these steps is crucial for a fully functional sharded keyspace. This guide will walk you through these steps, with a focus on VSchema definition and usage.
+
+ {{< info >}}
+ **Note**: Applying the VSchema (step 2) only defines the logical structure of your sharded keyspace. It does not create physical shards, tablets, or initialize the schema. Attempting to query the database after only applying the VSchema will result in errors such as "no healthy tablet available". Ensure you complete all the steps listed above before attempting to use your sharded keyspace.
+{{< /info >}}
```
Comment (mattlord) [inline content/en/docs/20.0/user-guides/vschema-guide/sharded.md:20]: I don't think that this helps resolve the issue. It attempts to explain Vitess concepts when that is done elsewhere (it's assumed that you have some basic concepts in mind before you ever get to this point). It is also a bit confusing to me as I'm not sure what a physical shard is, how that is that not made up of tablets and mysqld pairs, etc.

I believe that the issue would be resolved by noting here that it's assumed you have run the local examples steps 101 - 307 and linking to the source code for those or other relevant docs.

Comment (Satarupa22-SD) [reply content/en/docs/20.0/user-guides/vschema-guide/sharded.md:20]: Thank you for pointing it out.  I will check the existing documentation and create a new PR.

**Pull request closed without merging**